src/Lecture2.hs

```diff
@@ -185,7 +241,9 @@ False
 True
 -}
 isIncreasing :: [Int] -> Bool
-isIncreasing = error "TODO"
+isIncreasing [] = True
+isIncreasing [x] = True
```

Comment:
You don't care about variable name here so you can use the underscore pattern here 
```suggestion
isIncreasing [_] = True
```